core/templates/domain/collection/collection-summary.model.spec.ts

```diff
@@ -12,20 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
 /**
- * @fileoverview Unit tests for collectio  Summary Object Factory.
+ * @fileoverview Unit tests for CollectionSummary.
  */
 
-import { CollectionSummaryObjectFactory } from
-  'domain/collection/collection-summary-object.factory';
+import { CollectionSummary } from './collection-summary.model';
 
 describe('Collection summary object factory', () => {
```

Comment:
We don't have any factory service so rename these test messages accordingly* (Follow this in all other files*)

Reply:
done